app/cache/SessionCache.scala

```diff
@@ -20,12 +20,17 @@ import play.api.libs.json._
 import uk.gov.hmrc.cache.model.Cache
 import uk.gov.hmrc.cache.repository.CacheMongoRepository
 
+import uk.gov.hmrc.mongo.cache.CacheItem
+import uk.gov.hmrc.mongo.cache.MongoCacheRepository
+import uk.gov.hmrc.mongo.cache.EntityCache
+
 import javax.inject.Singleton
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 trait SessionCache[A] {
-  this: CacheMongoRepository =>
+  //this: MongoCacheRepository =>
```

Comment:
It's unclear from here: https://github.com/hmrc/hmrc-mongo/blob/main/MIGRATION.md#cache
How to resolve this session cache issue at this time.